### DISCUSSION TITLE
Is it possible to have HTTPS between the reverse proxy and the Nextcloud All-in-One (AIO) setup?

### DISCUSSION THREAD

**Discussion by redyolocraft** [nextcloud/all-in-one#3559] [Questions] [answered]
Hello, I have set up Nextcloud All-in-One in a separate virtual machine just for it. This virtual machine has the IP address 192.168.0.26, and I have another virtual machine with a reverse proxy with the IP address 192.168.0.6. I configured Nextcloud All-in-One as described here (https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md). However, the communication between the reverse proxy and Nextcloud is done over HTTP, which bothers me. Is there a way to have it use HTTPS between the reverse proxy and Nextcloud, even if it's self-signed?

**Comment by szaimen** [comment, marked as answer]
Hi, see 
![image](https://github.com/nextcloud/all-in-one/assets/42591237/416e7cb4-159c-47ba-a96e-50c52a7822a5)
